interpreter/per-host/src/main/scala/io/buoyant/transformer/perHost/Ec2HostTransformerInitializer.scala

```diff
@@ -0,0 +1,36 @@
+package io.buoyant.transformer
+package perHost
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import io.buoyant.namer.{NameTreeTransformer, TransformerConfig, TransformerInitializer}
+import java.net.InetAddress
+import com.twitter.finagle.{Http, Stack, http, Path}
+import com.twitter.util.Await
+
+class Ec2HostTransformerInitializer extends TransformerInitializer {
+  val configClass = classOf[Ec2HostTransformerConfig]
+  override val configId = "io.l5d.ec2Host"
+}
+
+class Ec2HostTransformerConfig extends TransformerConfig {
+
+  @JsonIgnore
+  val defaultPrefix = Path.read("/io.l5d.ec2Host")
+
+  @JsonIgnore
+  override def mk(params: Stack.Params): NameTreeTransformer = {
+
+    val servicePort = 80
+    val serviceHost = "169.254.169.254"
```

Comment:
The AWS API always lives at this address.  It is a special endpoint that you can hit on any AWS EC2 box.  I don't believe this is possible to change so I'm not sure it makes sense to make this configurable?